plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -6,32 +6,36 @@ library ExitPriority {
 
     using PosLib for PosLib.Position;
 
+    uint8 constant internal SIZEOF_TIMESTAMP = 32;
+    uint8 constant internal SIZEOF_EXITID = 168;
+
     /**
      * @dev Returns an exit priority for a given UTXO position and a unique ID.
      * The priority for Plasma M(ore)VP protocol is a combination of 'exitableAt' and 'txPos'.
-     * Since 'exitableAt' only provides granularity of block, add 'txPos' to provide priority for a transaction.
+     * Since 'exitableAt' only provides granularity of block, adding 'txPos' gives higher priority to transactions added earlier in the same block.
      * @notice Detailed explanation on field lengths can be found at https://github.com/omisego/plasma-contracts/pull/303#discussion_r328850572
```

Comment:
consider to move the exitId discussion from security-issue to plasma-contract repo and link it here for future code reader too